exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -106,11 +105,19 @@
   String RETURN_ERROR_FOR_FAILURE_IN_CANCELLED_FRAGMENTS =
       "drill.exec.debug.return_error_for_failure_in_cancelled_fragments";
 
+  String CLIENT_SUPPORT_COMPLEX_TYPES = "drill.client.supports-complex-types";
 
+  /**
+   * Configuration properties connected with dynamic UDFs support
+   */
```

Comment:
Are these only for dynamic UDFs? Do we have settings for (non-dynamic) UDFs? If so, should we have

drill.exec.udf
   prop1
   dynamic
      prop1
      prop2